ext/shmop/php_shmop.stub.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
File name should be `shmop.stub.php` to be consistent with the using file `shmop.c`.